<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UB6TGO0" failures="1" tests="2" name="cucumber.runners.TestNG_Test" time="28.791" errors="0" timestamp="2023-04-17T16:27:25 PDT" skipped="0">
  <testcase classname="cucumber.runners.TestNG_Test" name="runScenario" time="6.932"/>
  <system-out/>
  <testcase classname="cucumber.runners.TestNG_Test" name="runScenario" time="21.859">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1030)
at org.testng.Assert.assertEquals(Assert.java:1076)
at cucumber.stepdefinition.withtestng.LoginTestNG_Definition.user_should_be_able_to_see_error_message(LoginTestNG_Definition.java:55)
at ✽.User should be able to see error message "The password you’ve entered is incorrect. Forgot Password?"(file:///C:/THAO/SourceCode/example/thaoproj/src/test/resources/feature/loginTestNG.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- cucumber.runners.TestNG_Test -->
